<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailWidget</class>
 <widget class="QWidget" name="DetailWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Học bạ chi tiết</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HỌC BẠ ĐIỆN TỬ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tên HS: </string>
   </property>
  </widget>
  <widget class="QLabel" name="TenHSLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>unknown</string>
   </property>
  </widget>
  <widget class="QLabel" name="SoGiaoDucLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>unknown</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>541</width>
     <height>192</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <column>
    <property name="text">
     <string>Môn học</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Lớp</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Điểm</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Xếp Loại</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Giáo viên</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
